openstack/compute/v2/extensions/evacuate/requests.go

```diff
@@ -0,0 +1,41 @@
+package evacuate
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// EvacuateOptsBuilder allows extensions to add additional parameters to the
+// the Evacuate request.
+type EvacuateOptsBuilder interface {
+	ToEvacuateMap() (map[string]interface{}, error)
+}
+
+// EvacuateOpts specifies Evacuate action parameters.
+type EvacuateOpts struct {
+	// The name of the host to which the server is evacuated
+	Host string `json:"host"`
```

Comment:
According to both the API reference and Nova code, `host` is optional, so I _think_ this should be `json:"host,omitempty"`.